### PR TITLE
Adds IpInIp to Felix config and alphabetizes tables

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Continues https://github.com/projectcalico/calico/pull/1431, which adds `IpInIpEnabled`.
- Adds `ipipEnabled` to the `calicoctl` Felix resource, as well.
- Corrects erroneous discrepancy between master and v3.0 for `failsafeInboundHostPorts` port 179: should be `tcp` in both.
- Updates `InterfacePrefix` in Felix configuration to show the new support for comma-delimited list.
- Alphabetizes the tables in Felix configuration and calicoctl Felix resource. No content changes except for the addition of `IpInIPEnabled`. 

NOTE: Apologies for the diffs which look more extensive than they are, but we really need to alphabetize these tables.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by emanic** [comment]
@fasaxc Do we need to add `IpInIPEnabled` to the `calicoctl` Felix resource as well?

File: v3.0/reference/felix/configuration.md [hunk 1]
```diff
@@ -66,30 +67,31 @@ The Kubernetes datastore driver reads its configuration from Kubernetes-provided
...
 
 
 #### Bare metal specific configuration
 
 | Configuration parameter | Environment variable    | Description | Schema |
 | ----------------------- | ----------------------- | ----------- | ------ |
-| `InterfacePrefix`       | `FELIX_INTERFACEPREFIX` | The interface name prefix that identifies workload endpoints and so distinguishes them from host endpoint interfaces.  Note: in environments other than bare metal, the orchestrators configure this appropriately.  For example our Kubernetes and Docker integrations set the `cali` value. [Default: `cali`] | string |
+| `InterfacePrefix`       | `FELIX_INTERFACEPREFIX` | The interface name prefix that identifies workload endpoints and so distinguishes them from host endpoint interfaces.  Note: in environments other than bare metal, the orchestrators configure this appropriately.  For example our Kubernetes and Docker integrations set the `cali` value, and our OpenStack integration sets the `tap` value. [Default: `cali`] | string |
```
Comment (fasaxc) [inline v3.0/reference/felix/configuration.md]: This parameter now accepts a comma-delimited list of prefixes.  (So we can support `tap` and `cali` at the same time.)

Comment (emanic) [reply v3.0/reference/felix/configuration.md]: OK thanks, I'll update this!

**Comment by emanic** [comment]
Oh good point, I will add that.

**Comment by fasaxc** [comment]
> Do we need to add IpInIPEnabled to the calicoctl Felix resource as well?

@emanic the calicoctl one refers to the values that are defined here in the `FelixConfigurationSpec`: https://github.com/projectcalico/libcalico-go/blob/master/lib/apis/v3/felixconfig.go

In that struct, the field is called `IPIPEnabled`  The discrepancy is unfortunate; we improved the naming in that struct but we didn't have enough time in v3.0 to bring Felix's names into sync.

**Pull request merged**